core/src/main/java/jenkins/model/JenkinsLocationConfiguration.java

```diff
@@ -50,6 +50,17 @@
     public static @CheckForNull JenkinsLocationConfiguration get() {
         return GlobalConfiguration.all().get(JenkinsLocationConfiguration.class);
     }
+    
+    /**
+     * Gets local configuration. For explanation when it could die, see {@link #get()}
+     */
+    public static @Nonnull JenkinsLocationConfiguration getOrDie(){
```

Comment:
Probably should add `@Restricted(NoExternalUse.class)` since it is a new public method.